instruqt/avoiding-installation-pitfalls/02-checking-cluster-resources/assignment.md

```diff
@@ -143,24 +143,18 @@ Now that we have a thorough set of preflights for cluster resources, let's run
 them:
 
 ```
-kubectl preflight ./harbor-preflights.yaml
+kubectl preflight ./slackernews-preflights.yaml
 ```
 
 You'll see that all three preflights are run, and that the memory
 preflight has failed. This is an expected failure, since we have
-single node cluster that uses just that node's disk for storage. That
-disk is smaller than the storage requirements for Harbor.
+single node cluster that has a small memory footprint.
```

Comment:
When I run this today, the memory failure is actually a warning:


```
   - nodeResources:
        checkName: Cluster memory is sufficient to install and run Slackernews
        outcomes:
          - fail:
              when: "sum(memoryAllocatable) < 4G"
              message: |-
                Slackernews requires a minimum of 4 GB of memory in order to run, and runs best with
                at least 8 GB. Your current cluster has less than 4 GB available to Kubernetes
                workloads. Please increase cluster capacity or install into a different cluster.
              uri: https://goslackernews.io/docs/2.8.0/install-config/installation-prereqs/
          - warn:
              when: "sum(memoryAllocatable) < 8Gi"
              message: |-
                Slackernews runs best with a minimum of 8 GB of memory. Your current cluster has less
                than 8 GB of memory available to run workloads. For the best experience, consider
                increasing cluster capacity or installing into a different cluster.
              uri: https://goslackernews.io/docs/2.8.0/install-config/installation-prereqs/
          - pass:
              message: Your cluster has sufficient memory available to run Slackernews

replicant@shell:~$ free -h
               total        used        free      shared  buff/cache   available
Mem:            12Gi       1.1Gi       2.0Gi        28Mi       9.6Gi        11Gi
Swap:             0B          0B          0B
replicant@shell:~$ 
```

Reply:
I think a warning actually makes sense based on Slackernews (was good as a failure for Harbor).

Reply:
But I'm a little bit uncertain right now what to change here. It actually looks like a pass.

Reply:
Sorted it out looking at the next bit of feedback.